test.js

```diff
@@ -41,7 +41,7 @@ const macro = async (t, options) => {
 
 test('default', macro, {});
 test('verbose option', macro, {verbose: true});
-test('filter option', macro, {filter: ['status eq running', 'username ne F4k3U53RN4M3']});
+test('filter option', macro, {filter: ['sessionname eq console', 'username ne F4k3U53RN4M3']});
```

Comment:
CI build server is reporting a status of `Unknown` for every process. There is a fair distribution between `console` and `service` session names.

Sample output: https://travis-ci.org/brandon93s/tasklist/jobs/473901717